aries-site/src/components/feedback/Question.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormField,
+  TextArea,
+  Text,
+  RadioButtonGroup,
+  FileInput,
+} from 'grommet';
+import { StarRating } from './StarRating';
+import { ThumbsRating } from './ThumbsRating';
+
+export const Question = ({ formProps, label, inputProps, kind, name }) => {
+  const accessibilityProps = {
+    id: name,
+    name,
+  };
+
+  const formats = {
+    star: {
+      render: StarRating,
+      border: false,
+      label: <Text size="large">{label}</Text>,
+    },
+    thumbs: {
+      render: ThumbsRating,
+      border: false,
+      label: <Text size="large">{label}</Text>,
+    },
+    textArea: { render: TextArea },
```

Comment:
I could be swayed in that direction. Wondering if we'd want to circle back with Megan on it?

I think it depends on the visual hierarchy we want in the modal. I had initially interpreted it as "primary question" + "secondary/supporting/aside question" and felt like the text hierarchy supported that. If we want the questions to feel like peers, then I would support having the text treatment match.

Reply:
I think this is another case of the restrictions of certain Figma components. People feel locked in to what the component currently does or just take the default behavior ...

I interpret the questions as peers. I can't think of any survey I've taken or written which has question hierarchy -- that doesn't mean it can't be done, rather I'd like to have a reason justifying it. Additionally, from a component standpoint, I'd think the component would default to a uniform question typography; we could introduce the concept of a child question, but I wouldn't assume the TextArea kind of question to be secondary, what if it were the first question?

Reply:
I can circle back with Megan on this I think as she was using the figma components she did not change the size of the text for the TextArea example which might be related to `. People feel locked in to what the component currently does or just take the default behavior `

I agree with you though in the fact that there are reviews I have written and the `text` was not different sizes or in any order